### PR TITLE
refactor button component, refine css for mobile

### PR DESCRIPTION
**Pull request by werner33**
The submit button is extracted out into a more generic component. 
Submit button is larger. < than 600px it fills the screen. 

Todo: make loader into icon and lay properly in button.

### CODE REVIEW

File: codeTykeReact/src/components/Button.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+
+const Button = (props) => {
+
+  let label = props.label;
+  let className = `submitButton ${props.type}`;
+  let icon = props.icon;
+
+  const loaderImg = <div className="submitLoader" style={{display: props.loading ? "block" : "none"}}><img src='loadingLogo.png' /></div>;
```
Comment (brightcodedev) [inline codeTykeReact/src/components/Button.js]: Minor: I wouldn't make variables of JSX unless it's reused somewhere else. It's better to keep JSX together

Comment (brightcodedev) [reply codeTykeReact/src/components/Button.js]: That wasn't from a change you made but something I'm noticing now that it's been moved

Comment (werner33) [reply codeTykeReact/src/components/Button.js]: Yes, you're right. I'll  change that now.

Comment (werner33) [reply codeTykeReact/src/components/Button.js]: Done.

**Pull request merged**